data/basics/userBalance.go

```diff
@@ -381,6 +381,14 @@ func (u AccountData) Money(proto config.ConsensusParams, rewardsLevel uint64) (m
 	return e.MicroAlgos, e.RewardedMicroAlgos
 }
 
+// PendingRewards computes the amount of rewards (in microalgos) that
+// have yet to be added to the account balance.
+func PendingRewards(ot *OverflowTracker, microAlgos MicroAlgos, rewardsBase uint64, proto config.ConsensusParams, rewardsLevel uint64) MicroAlgos {
```

Comment:
nit: maybe move the `proto config.ConsensusParams` to be one of the first two parameters ? it's really weird that it's placed in the middle of the uint64 parameters.
(the function is perfectly fine otherwise )

Reply:
Yeah, that would be clearer indeed; thanks!